systemtest/src/test/java/io/strimzi/systemtest/kafka/QuotasST.java

```diff
@@ -84,7 +85,14 @@ void testKafkaQuotasPluginIntegration(ExtensionContext extensionContext) {
 
     @BeforeAll
     void setup(ExtensionContext extensionContext) {
-        installClusterWideClusterOperator(extensionContext, NAMESPACE, Constants.CO_OPERATION_TIMEOUT_DEFAULT, Constants.RECONCILIATION_INTERVAL);
+        install = new SetupClusterOperator.SetupClusterOperatorBuilder()
+            .withExtensionContext(extensionContext)
+            .withClusterOperatorName(Constants.STRIMZI_DEPLOYMENT_NAME)
```

Comment:
Do we need to specify it everywhere, when the name is almost always the same? (same for other classes)

Reply:
good catch